sqlflow_models/deep_embedding_cluster.py

```diff
@@ -272,6 +268,48 @@ def display_model_info(self, verbose=0):
             print(self.clustering_layer.name + ' : ')
             print(self.clustering_layer.get_weights())
 
+def get_model(feature_columns,
+              n_clusters=10,
+              kmeans_init=20,
+              run_pretrain=True,
+              existed_pretrain_model=None,
+              pretrain_dims=None,
+              pretrain_activation_func='relu',
+              pretrain_batch_size=256,
+              train_batch_size=256,
+              pretrain_epochs=10,
+              pretrain_initializer='glorot_uniform',
+              pretrain_lr=1,
+              train_lr=0.01,
+              train_max_iters=8000,
+              update_interval=100,
+              tol=0.001,
+              loss=kld):
+    return DeepEmbeddingClusterModel(feature_columns,
+        n_clusters=n_clusters,
+        kmeans_init=kmeans_init,
+        run_pretrain=run_pretrain,
+        existed_pretrain_model=existed_pretrain_model,
+        pretrain_dims=pretrain_dims,
+        pretrain_activation_func=pretrain_activation_func,
+        pretrain_batch_size=pretrain_batch_size,
+        train_batch_size=train_batch_size,
+        pretrain_epochs=pretrain_epochs,
+        pretrain_initializer=pretrain_initializer,
+        pretrain_lr=pretrain_lr,
+        train_lr=train_lr,
+        train_max_iters=train_max_iters,
+        update_interval=update_interval,
+        tol=tol,
+        loss=loss)
+
+def optimizer():
+    global _train_lr
```

Comment:
Why are these globals needed?

Reply:
`_train_lr` may be configured by SQL statements when initializing the model instance. e.g.

```
SELECT * FROM mytrain_table TO TRAIN sqlflow_models.DNNClassifier WITH train_lr=0.0001 ...
```

In the above SQL statement, `train_lr` will be passed to `DeepEmbeddingClusterModel` to configure the learning rate.